tests/lib/eslint/flat-eslint.js

```diff
@@ -2607,7 +2607,17 @@ describe("FlatESLint", () => {
             let id;
 
             beforeEach(() => (id = Date.now().toString()));
-            afterEach(async () => fsp.rmdir(root, { recursive: true, force: true }));
+
+            /*
+             * `fs.rmdir(path, { recursive: true })` is deprecated and will be removed.
+             * Use `fs.rm(path, { recursive: true })` instead.
+             * When supporting Node.js 14+, the compatibility condition can be removed for `fs.rmdir`.
```

Comment:
```suggestion
             * When supporting Node.js 14.14.0+, the compatibility condition can be removed for `fs.rmdir`.
```

According to the [documentation](https://nodejs.org/api/fs.html#fspromisesrmpath-options), we'll need 14.14+ to remove this condition?

> __fsPromises.rm(path[, options]))__
> Added in: v14.14.0

Reply:
Ah, you're right! I've confirmed that and updated this PR.